Source: written by Benedikt Morbach <benedikt.morbach@googlemail.com>
Upstream: submitted, http://bugs.exim.org/show_bug.cgi?id=1220
Reason: fix multilib install, which needs those symlinks to be overwritten

--- a/Makefile.am	2012-01-27 12:10:10.000000000 +0100
+++ b/Makefile.am	2012-03-23 11:48:53.488654246 +0100
@@ -521,31 +521,31 @@
 
 # Arrange for the per-function man pages to have 16-bit names as well.
 install-data-hook:
-	ln -s pcre_assign_jit_stack.3		$(DESTDIR)$(man3dir)/pcre16_assign_jit_stack.3
-	ln -s pcre_compile.3			$(DESTDIR)$(man3dir)/pcre16_compile.3
-	ln -s pcre_compile2.3			$(DESTDIR)$(man3dir)/pcre16_compile2.3
-	ln -s pcre_config.3			$(DESTDIR)$(man3dir)/pcre16_config.3
-	ln -s pcre_copy_named_substring.3	$(DESTDIR)$(man3dir)/pcre16_copy_named_substring.3
-	ln -s pcre_copy_substring.3		$(DESTDIR)$(man3dir)/pcre16_copy_substring.3
-	ln -s pcre_dfa_exec.3			$(DESTDIR)$(man3dir)/pcre16_dfa_exec.3
-	ln -s pcre_exec.3			$(DESTDIR)$(man3dir)/pcre16_exec.3
-	ln -s pcre_free_study.3			$(DESTDIR)$(man3dir)/pcre16_free_study.3
-	ln -s pcre_free_substring.3		$(DESTDIR)$(man3dir)/pcre16_free_substring.3
-	ln -s pcre_free_substring_list.3	$(DESTDIR)$(man3dir)/pcre16_free_substring_list.3
-	ln -s pcre_fullinfo.3			$(DESTDIR)$(man3dir)/pcre16_fullinfo.3
-	ln -s pcre_get_named_substring.3	$(DESTDIR)$(man3dir)/pcre16_get_named_substring.3
-	ln -s pcre_get_stringnumber.3		$(DESTDIR)$(man3dir)/pcre16_get_stringnumber.3
-	ln -s pcre_get_stringtable_entries.3	$(DESTDIR)$(man3dir)/pcre16_get_stringtable_entries.3
-	ln -s pcre_get_substring.3		$(DESTDIR)$(man3dir)/pcre16_get_substring.3
-	ln -s pcre_get_substring_list.3		$(DESTDIR)$(man3dir)/pcre16_get_substring_list.3
-	ln -s pcre_jit_stack_alloc.3		$(DESTDIR)$(man3dir)/pcre16_jit_stack_alloc.3
-	ln -s pcre_jit_stack_free.3		$(DESTDIR)$(man3dir)/pcre16_jit_stack_free.3
-	ln -s pcre_maketables.3			$(DESTDIR)$(man3dir)/pcre16_maketables.3
-	ln -s pcre_pattern_to_host_byte_order.3 $(DESTDIR)$(man3dir)/pcre16_pattern_to_host_byte_order.3
-	ln -s pcre_refcount.3			$(DESTDIR)$(man3dir)/pcre16_refcount.3
-	ln -s pcre_study.3			$(DESTDIR)$(man3dir)/pcre16_study.3
-	ln -s pcre_utf16_to_host_byte_order.3	$(DESTDIR)$(man3dir)/pcre16_utf16_to_host_byte_order.3
-	ln -s pcre_version.3			$(DESTDIR)$(man3dir)/pcre16_version.3
+	ln -sf pcre_assign_jit_stack.3		$(DESTDIR)$(man3dir)/pcre16_assign_jit_stack.3
+	ln -sf pcre_compile.3			$(DESTDIR)$(man3dir)/pcre16_compile.3
+	ln -sf pcre_compile2.3			$(DESTDIR)$(man3dir)/pcre16_compile2.3
+	ln -sf pcre_config.3			$(DESTDIR)$(man3dir)/pcre16_config.3
+	ln -sf pcre_copy_named_substring.3	$(DESTDIR)$(man3dir)/pcre16_copy_named_substring.3
+	ln -sf pcre_copy_substring.3		$(DESTDIR)$(man3dir)/pcre16_copy_substring.3
+	ln -sf pcre_dfa_exec.3			$(DESTDIR)$(man3dir)/pcre16_dfa_exec.3
+	ln -sf pcre_exec.3			$(DESTDIR)$(man3dir)/pcre16_exec.3
+	ln -sf pcre_free_study.3			$(DESTDIR)$(man3dir)/pcre16_free_study.3
+	ln -sf pcre_free_substring.3		$(DESTDIR)$(man3dir)/pcre16_free_substring.3
+	ln -sf pcre_free_substring_list.3	$(DESTDIR)$(man3dir)/pcre16_free_substring_list.3
+	ln -sf pcre_fullinfo.3			$(DESTDIR)$(man3dir)/pcre16_fullinfo.3
+	ln -sf pcre_get_named_substring.3	$(DESTDIR)$(man3dir)/pcre16_get_named_substring.3
+	ln -sf pcre_get_stringnumber.3		$(DESTDIR)$(man3dir)/pcre16_get_stringnumber.3
+	ln -sf pcre_get_stringtable_entries.3	$(DESTDIR)$(man3dir)/pcre16_get_stringtable_entries.3
+	ln -sf pcre_get_substring.3		$(DESTDIR)$(man3dir)/pcre16_get_substring.3
+	ln -sf pcre_get_substring_list.3		$(DESTDIR)$(man3dir)/pcre16_get_substring_list.3
+	ln -sf pcre_jit_stack_alloc.3		$(DESTDIR)$(man3dir)/pcre16_jit_stack_alloc.3
+	ln -sf pcre_jit_stack_free.3		$(DESTDIR)$(man3dir)/pcre16_jit_stack_free.3
+	ln -sf pcre_maketables.3			$(DESTDIR)$(man3dir)/pcre16_maketables.3
+	ln -sf pcre_pattern_to_host_byte_order.3 $(DESTDIR)$(man3dir)/pcre16_pattern_to_host_byte_order.3
+	ln -sf pcre_refcount.3			$(DESTDIR)$(man3dir)/pcre16_refcount.3
+	ln -sf pcre_study.3			$(DESTDIR)$(man3dir)/pcre16_study.3
+	ln -sf pcre_utf16_to_host_byte_order.3	$(DESTDIR)$(man3dir)/pcre16_utf16_to_host_byte_order.3
+	ln -sf pcre_version.3			$(DESTDIR)$(man3dir)/pcre16_version.3
 
 pcrecpp_man = doc/pcrecpp.3
 EXTRA_DIST += $(pcrecpp_man)
